Vision/SplitImage/README.md

```diff
@@ -5,12 +5,13 @@ languages:
 products:
 - azure
 - azure-search
-azureDeploy: https://raw.githubusercontent.com/Azure-Samples/azure-search-power-skills/master/Text/SplitImage/azuredeploy.json
+azureDeploy: https://raw.githubusercontent.com/Azure-Samples/azure-search-power-skills/master/Vision/SplitImage/azuredeploy.json
 name: "Split Image sample skill for Azure Cognitive Search"
 description: "This Split Image skill can be used on images that are too long to process in the vanilla pipeline. This skill will break a large image up into several images small enough to be processed by OCR."
 ---
 
 # Split Image
+IMPORTANT: This skill uses external, unverified nuget packages!! Use with caution, and do not use with sensitive data!
```

Comment:
Suggested rewording:

"DISCLAIMER: This skill uses a third-party package, `TiffLibrary.ImageSharpAdapter` [add link]. Use at your own risk. If you need to verify the source code of this package, you can find its source code here: [add link], and verify the contents of the package corresponds to the source in that repository using publicly available tools."